linux_os/guide/system/accounts/accounts-restrictions/account_expiration/ensure_shadow_group_empty/rule.yml

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Ensure shadow group is empty'
+
+description: |-
+    The shadow group allows system programs which require access the ability 
+    to read the /etc/shadow file. No users should be assigned to the shadow group.
+
+rationale: |-
+    Any users assigned to the shadow group would be granted read access to the 
+    /etc/shadow file. If attackers can gain read access to the /etc/shadow file, 
+    they can easily run a password cracking program against the hashed passwords 
+    to break them. Other security information that is stored in the /etc/shadow 
+    file (such as expiration) could also be useful to subvert additional user 
+    accounts.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92213-8
+    cce@sle15: CCE-91344-2
```

Comment:
Please remove the used CCE from shared/references/cce-sle12-avail.txt and shared/references/cce-sle15-avail.txt.